app/src/main/java/org/sopt/and/services/UserService.kt

```diff
@@ -0,0 +1,22 @@
+package org.sopt.and.services
+
+import org.sopt.and.myinfo.dto.GetHobbyResponseDto
+import org.sopt.and.signin.dto.SignInRequestDto
+import org.sopt.and.signin.dto.SignInResponseDto
+import org.sopt.and.signup.dto.SignUpRequestDto
+import org.sopt.and.signup.dto.SignUpResponseDto
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+
+interface UserService {
+    @POST("/user")
+    fun signUp(@Body request: SignUpRequestDto): Call<SignUpResponseDto>
+
+    @POST("/login")
+    fun signIn(@Body request: SignInRequestDto): Call<SignInResponseDto>
+
+    @GET("/user/my-hobby")
+    fun getMyHobby(): Call<GetHobbyResponseDto>
```

Comment:
call 객체를 반환하기 보다 suspend 로 인터페이스를 변경하심을 추천드려요. (이유는 별도 코멘트)
```suggestion
    @POST("/user")
    suspend fun signUp(@Body request: SignUpRequestDto): SignUpResponseDto

    @POST("/login")
    suspend fun signIn(@Body request: SignInRequestDto): SignInResponseDto

    @GET("/user/my-hobby")
    suspend fun getMyHobby(): GetHobbyResponseDto
```